services/wallet/storage/storage_test.go

```diff
@@ -0,0 +1,318 @@
+//go:build integration
+
+package storage
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/brave-intl/bat-go/libs/datastore"
+	"github.com/brave-intl/bat-go/services/wallet/model"
+	"github.com/jmoiron/sqlx"
+	uuid "github.com/satori/go.uuid"
+	must "github.com/stretchr/testify/require"
+)
+
+func TestChallenge_Get(t *testing.T) {
+	dbi, err := setupDBI()
+	must.NoError(t, err)
+
+	defer func() {
+		_, _ = dbi.Exec("TRUNCATE_TABLE challenge;")
+	}()
+
+	type tcGiven struct {
+		id   string
+		chal model.Challenge
+	}
+
+	type exp struct {
+		chal model.Challenge
+		err  error
+	}
+
+	type testCase struct {
+		name  string
+		given tcGiven
+		exp   exp
+	}
+
+	tests := []testCase{
+		{
+			name: "get",
+			given: tcGiven{
+				id: "1",
+				chal: model.Challenge{
+					ID:        "1",
+					CreatedAt: time.Date(2024, 1, 1, 1, 1, 1, 0, time.UTC),
+					Nonce:     "nonce-1",
+				}},
+			exp: exp{
+				chal: model.Challenge{
+					ID:        "1",
+					CreatedAt: time.Date(2024, 1, 1, 1, 1, 1, 0, time.UTC),
+					Nonce:     "nonce-1",
+				},
+				err: nil,
+			},
+		},
+		{
+			name: "not_found",
+			given: tcGiven{
+				id: "some-random-id",
+				chal: model.Challenge{
+					ID:        "2",
+					CreatedAt: time.Date(2024, 1, 1, 1, 1, 1, 0, time.UTC),
+					Nonce:     "nonce-2",
+				}},
+			exp: exp{
+				err: model.ErrNotFound,
+			},
+		},
+	}
+
+	const q = `insert into challenge (id, created_at, nonce) values($1, $2, $3)`
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			ctx := context.Background()
+
+			_, err1 := dbi.ExecContext(ctx, q, tc.given.chal.ID, tc.given.chal.CreatedAt, tc.given.chal.Nonce)
+			must.NoError(t, err1)
+
+			c := Challenge{}
+			actual, err2 := c.Get(ctx, dbi, tc.given.id)
+			must.Equal(t, tc.exp.err, err2)
+
+			must.Equal(t, tc.exp.chal.ID, actual.ID)
+			must.Equal(t, tc.exp.chal.CreatedAt, actual.CreatedAt)
+			must.Equal(t, tc.exp.chal.Nonce, actual.Nonce)
```

Comment:
`must` (aka `require`) is a show stopper and will fail the entire test immediately. We want this behaviour for unexpected situations, such as unexpected failures. It serves more or less as `if err != nil {}`.

`should` (aka `assert`) is for regular assertions. If we use `must` here, we would fail the entire test without finishing all test cases, which is not a desired behaviour.

```suggestion
			should.Equal(t, tc.exp.chal.ID, actual.ID)
			should.Equal(t, tc.exp.chal.CreatedAt, actual.CreatedAt)
			should.Equal(t, tc.exp.chal.Nonce, actual.Nonce)
```

Reply:
This is due to the IDE doing the naming conversion between the alias i.e. must and require and should and assert. When using auto complete for tests the renaming is causing extra work as it needs to add the alias. 

It seems we have adopted the should and must in other parts as the code base although the majority of the tests use require and assert. Maybe we should discuss which direction we want to go i.e. continue with the assert/require or make the switch to using should/must and what the benefit would be.